packages/ra-ui-materialui/src/list/ListView.tsx

```diff
@@ -100,7 +100,7 @@ export const ListView: FC<ListViewProps> = props => {
             {...sanitizeRestProps(rest)}
         >
             <Title title={title} defaultTitle={defaultTitle} />
-            {shouldRenderEmptyPage
+            {shouldRenderEmptyPage && empty !== false
```

Comment:
Why not update the `shouldRenderEmptyPage` method to use the `empty` prop?

Reply:
Because of Typescript ;)